octree_web_viewer/src/state.rs

```diff
@@ -47,6 +50,7 @@ impl AppState {
                 suffix: suffix.into(),
             },
             init_octree_id: octree_id.into(),
+            octree_factory: input_octree_factory,
```

Comment:
once you renamed the parameter, you can write `octree_factory,` instead of `octree_factory: input_octree_factory`

Reply:
thanks for the review and suggestion!